src/main/java/fruit/shop/model/Transaction.java

```diff
@@ -0,0 +1,5 @@
+package fruit.shop.model;
+
+public enum Transaction {
```

Comment:
The instance of this class won't be transaction - `TransactionType` would be more descriptive.